irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -30,29 +30,20 @@ namespace iroha {
         const model::Proposal &proposal,
         ametsuchi::TemporaryWsv &temporaryWsv) {
       log_->info("transactions in proposal: {}", proposal.transactions.size());
-      auto checking_transaction = [&temporaryWsv](auto &tx, auto &executor,
-                                                  auto &query) {
+      auto checking_transaction = [&temporaryWsv](auto &tx, auto &query) {
         auto account = temporaryWsv.getAccount(tx.creator_account_id);
         // Check if tx creator has account and has quorum to execute transaction
         if (!account || tx.signatures.size() < account.value().quorum)
           return false;
 
         // Check if signatures in transaction are account signatory
         auto account_signs = temporaryWsv.getSignatories(tx.creator_account_id);
-        if (!account_signs)
+        if (not account_signs)
           // No signatories found
```

Comment:
I suggest creating a separate issue for this task and merge this code, if it looks good @grimadas @muratovv